.circleci/local_publish_helpers.sh

```diff
@@ -12,6 +12,36 @@ function startLocalRegistry {
     grep -q 'http address' <(tail -f $tmp_registry_log)
 }
 
+function setNpmTag {
+    if [ -z $NPM_TAG ]; then
+        if [[ "$CIRCLE_BRANCH" =~ ^tagged-release ]]; then
+            if [[ "$CIRCLE_BRANCH" =~ ^tagged-release-without-e2e-tests\/.* ]]; then
+                export NPM_TAG="${CIRCLE_BRANCH/tagged-release-without-e2e-tests\//}"
+            elif [[ "$CIRCLE_BRANCH" =~ ^tagged-release\/.* ]]; then
+                export NPM_TAG="${CIRCLE_BRANCH/tagged-release\//}"
+            fi
+        fi
+    fi
+}
+
+function uploadPkgCli {
+    cd out/
+    export hash=$(git rev-parse HEAD | cut -c 1-12)
+    export version=$(./amplify-pkg-linux --version)
+    sudo apt-get update
+    sudo apt-get install -y sudo tcl expect zip lsof jq groff python python-pip libpython-dev
+    sudo pip install awscli
+    aws s3 cp amplify-pkg-win.exe s3://pkg-cli-ci-do-not-delete/$(echo $version)/amplify-pkg-win-$(echo $hash).exe && aws s3 cp amplify-pkg-macos s3://pkg-cli-ci-do-not-delete/$(echo $version)/amplify-pkg-macos-$(echo $hash) && aws s3 cp amplify-pkg-linux s3://pkg-cli-ci-do-not-delete/$(echo $version)/amplify-pkg-linux-$(echo $hash)
+
+    if [ -z "$NPM_TAG" ]; then
+        exit 0
+    fi
+
+    echo "Tag name is $NPM_TAG. Uploading to s3://pkg-cli-ci-do-not-delete/$(echo $version)"
+    aws s3 cp amplify-pkg-win.exe s3://pkg-cli-ci-do-not-delete/$(echo $version)/amplify-pkg-win.exe && aws s3 cp amplify-pkg-macos s3://pkg-cli-ci-do-not-delete/$(echo $version)/amplify-pkg-macos && aws s3 cp amplify-pkg-linux s3://pkg-cli-ci-do-not-delete/$(echo $version)/amplify-pkg-linux
```

Comment:
deployment without hash (`s3://pkg-cli-ci-do-not-delete/$(echo $version)/amplify-pkg-macos`) is what deployed npm versions will attempt to install.